sub/filter_sdh.c

```diff
@@ -67,18 +67,22 @@ static inline int append(struct sd_filter *sd, struct buffer *buf, char c)
 
 static int get_char_bytes(char *str)
 {
-    // In case the final character is non-ASCII.
+    // In case the first character is non-ASCII.
     // Will only work with UTF-8 but you shouldn't be
     // using anything else anyway.
     if (str && str[0]) {
         if (!(str[0] >> 7 & 1)) {
-            return MPMIN(strlen(str), 1);
+            int bytes = strnlen(str, 1);
```

Comment:
No need for MPMIN. `strnlen` will return at most maxlen.

Reply:
Duh.